### ISSUE TITLE
Consolidate Config and utilize soft coding of variables

### ISSUE THREAD

**Issue by qCzar** [hellos3b/sjbha-bot#19]
After all these bugs related to the way the bot handles time, I decided to poke my head in to see how you addressed it and noticed so many times where you [hard coded the timezone into the bot](https://github.com/hellos3b/sjbha-bot/commit/d8185205fa72514071dab6d1a0dd218dd1d9e962). I think it'd be smarter to soft code that in using variables which are pulled from a central config.

So if the bot is ever forked and used on another cities discord, they just need to adjust the timezone from the config and be good to go. You can also throw in the ID's for channels into this config, as well as other settings that are hardcoded in. 

Basically my gist is to use a central config to maintain aspects of the bot that are expected to change.

**Comment by hellos3b** [comment]
yeah I have to rethink how to handle timezones because right now it's a hacky mess

I have ideas to redo a lot of the bot, might try to find a solution then

**Comment by hellos3b** [comment]
Timezone is now set as a global in `main.ts`
